borg/archive.py

```diff
@@ -650,7 +677,7 @@ def init_chunks(self):
     def report_progress(self, msg, error=False):
         if error:
             self.error_found = True
-        print(msg, file=sys.stderr if error else sys.stdout)
+        logger.log(logging.ERROR if error else logging.WARNING, msg)
```

Comment:
i have returned to report_progress() logging warnings or errors, that is, put my original change back in, to fix tests that were failing.

I believe that we should just stop using report_progress in general and have a specific function to manage errors. Right now, report_progress records errors *and* prints a message: we should just let it record an error and have the caller decide the proper log level.